src/features/hotspots/settings/HotspotSettings.tsx

```diff
@@ -150,9 +152,22 @@ const HotspotSettings = ({ hotspot }: Props) => {
     setNextState('discoveryMode')
   }, [setNextState])
 
-  const onPressUpdateHotspot = useCallback(() => {
-    setNextState('updateHotspot')
-  }, [setNextState])
+  const onPressUpdateHotspot = useCallback(async () => {
+    // Check for pending assert
+    const pendingTxns = await getAccountTxnsList('pending')
+    const txns = (await pendingTxns?.take(20)) as PendingTransaction[]
+    const hasPending = txns?.some(
```

Comment:
Little nitpick... I think `pendingTxns?.find()` would be a better fit because it would short circuit when it finds the first one.